constants/social.js

```diff
@@ -76,16 +76,34 @@ export const SHARE_NETWORKS = [
 ];
 
 export const PODCASTS_PLATFORMS = [
+  {
+    id: 'applepodcasts',
+    label: 'Apple Podcasts',
+    link:
+      'https://podcasts.apple.com/by/podcast/wir-by-беларуская-і-сусветная-культура/id1511316106',
+    // https://marketing.services.apple/apple-podcasts-identity-guidelines
+    badgeLink:
+      'https://res.cloudinary.com/wir-by/image/upload/c_scale,h_100,f_auto,q_auto/v1716138382/production/podcast-platforms/apple.png',
+    buildEpisodeLink: episodeId =>
+      `https://podcasts.apple.com/by/podcast/wir-by-беларуская-і-сусветная-культура/id1511316106?i=${episodeId}`,
+  },
   {
     id: 'yandexmusic',
     label: 'Yandex Music',
     link: 'https://music.yandex.ru/album/10622616',
+    // https://yandex.ru/support/music/button.html
+    badgeLink:
+      'https://res.cloudinary.com/wir-by/image/upload/c_scale,h_100,f_auto,q_auto/v1716139098/production/podcast-platforms/yandex.png',
+    episodeLink: episodeId => `https://music.yandex.com/album/10622616/track/${episodeId}`,
   },
   {
-    id: 'applepodcasts',
-    label: 'Apple Podcasts',
-    link:
-      'https://podcasts.apple.com/by/podcast/wir-by-беларуская-і-сусветная-культура/id1511316106',
+    id: 'spotifypodcasts',
+    label: 'Spotify',
+    link: 'https://open.spotify.com/show/61Q5hmZfpw3fDoNM6cGir5',
+    badgeLink:
+      'https://res.cloudinary.com/wir-by/image/upload/c_scale,h_100,f_auto,q_auto/v1716139336/production/podcast-platforms/spotify.png',
+    episodeLink: episodeId =>
+      `https://open.spotify.com/episode/6nQzNc5lJPXtQbsovlwqiI?si=${episodeId}`,
```

Comment:
I think it should be like this:

```suggestion
      `https://open.spotify.com/episode/${episodeId}`,
```

because `6nQzNc5lJPXtQbsovlwqiI` - this is only `Дзе жывуць душы беларусаў`:
https://open.spotify.com/episode/6nQzNc5lJPXtQbsovlwqiI

Reply:
if you share a specific episode -- the link is something like https://open.spotify.com/episode/6nQzNc5lJPXtQbsovlwqiI?si=3ad5d21211944ff7

Reply:
Ok I agree `si=` is something else, links are just https://open.spotify.com/episode/71T8FcWSlaGPQy9ZrtroVq or https://open.spotify.com/episode/32Sc9QezeVlAmKQg7ZFXcS

Fixed that.